enhancements/ingress/gcp-lb-global-access.md

```diff
@@ -0,0 +1,223 @@
+---
+title: gcp-internal-load-balancer-global-access
+authors:
+  - "@sgreene570"
+reviewers:
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+  - "@miciah"
+  - "@candita"
+  - "@rfredette"
+  - "@miheer"
+approvers:
+  - "@danehans"
+  - "@frobware"
+  - "@knobunc"
+  - "@miciah"
+creation-date: 2021-02-09
+last-updated: 2021-02-09
+status: implementable
+---
+
+# GCP Internal Ingress Load Balancer Global Access Option
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement extends the IngressController API to allow the user to enable
+the "Global Access" option for Ingress Controllers exposed via an Internal Load Balancer
+on GCP.
+
+## Motivation
+
+On GCP, Ingress Controllers that specify a `LoadBalancerScope` of `InternalLoadBalancer` are only externally accessible
+when a client is in the same VPC network and Google Cloud Region as the provider's Load Balancer. GCP exposes the
```

Comment:
What does "the provider" refer to in this context?  Does it matter, or could we remove "provider's"?  
```suggestion
when a client is in the same VPC network and Google Cloud Region as the Load Balancer. GCP exposes the
```

Reply:
We can remove "provider's". Thanks!